crates/mun_runtime/src/reflection.rs

```diff
@@ -90,32 +87,20 @@ macro_rules! impl_primitive_type {
             impl ReturnTypeReflection for $ty {
                 type Marshalled = Self;
 
-                fn type_name() -> &'static str {
-                    Self::type_info().name()
-                }
-
                 fn type_guid() -> abi::Guid {
                     Self::type_info().guid
                 }
+
+                fn type_name() -> &'static str {
+                    Self::type_info().name()
+                }
             }
         )+
     }
 }
 
 impl_primitive_type!(i8, i16, i32, i64, isize, u8, u16, u32, u64, usize, f32, f64, bool);
 
-impl ArgumentReflection for () {
```

Comment:
We currently don't support `()` (or empty) as an argument type, so I removed it.